nireports/reportlets/mosaic.py

```diff
@@ -57,6 +58,39 @@
 from nireports.tools.ndimage import load_api, rotate_affine, rotation2canonical
 
 
+def _create_listed_colormap_with_alpha(cmap_name: str, max_alpha: float = 0.75) -> ListedColormap:
```

Comment:
Just trying to run this function raises an exception. We need tests, and we need to verify that this creates the same thing, or something close enough. I would suggest a test like:

```python
def test_create_cmap():
    orig_cmap = ...
    listed_cmap = _create_listed_colormap_with_alpha("Reds")

    listed_cmap._init()
    assert np.allclose(orig_cmap._lut, listed_cmap._lut)
```

Reply:
Commit c42dad4. Resolving.